projects/documentation/content/migrations/2024-10-01 (1.0.0).md

```diff
@@ -0,0 +1,189 @@
+---
+layout: guide.njk
+title: 'Migration Guide: Spectrum Web Components (v1.0.0)'
+displayName: 2024/10/01 (v1.0.0)
+slug: migration-guide
+---
+
+## Migration: 2024/10/01 (v1.0.0)
+
+With the release of Spectrum Web Components version 1.0.0, we removed previously deprecated components and features.
+Starting with this version, Spectrum Web Components adheres to [semantic versioning (semver)](https://semver.org/), ensuring that any future breaking changes will be introduced only in major version updates. These changes will be communicated in advance, allowing developers to prepare and avoid disruption to existing applications. Patch releases, as part of this versioning approach, will be non-breaking and focus on fixes and minor improvements.
+
+If you're upgrading from an earlier version, please review the following changes carefully to ensure your project remains compatible.
+
+## Removed Components and Features
+
+### 1. Banner (`sp-banner`)
+
+The `sp-banner` component has been removed.
+
+**Action Required**: If you are using `sp-banner`, remove it from your application and replace it with the [alert banner](/components/alert-banner/) component to show pressing and high-signal messages, such as system alerts.
+
+### 2. Quick actions (`sp-quick-actions`)
+
+The `sp-quick-actions` component has been removed.
+**Action Required**: If you are using `sp-quick-actions`, remove it from your application and replace it with the [action bar](/components/alert-banner/) component to allow users to perform actions on either a single or multiple items at the same time.
+
+### 3. Split button (`sp-split-button`)
+
+The `sp-split-button` component has been removed.
+**Action Required**: If you are using `sp-split-button`, remove it from your application and replace it with the [button group](/components/button-group/) component to to show any additional actions related to the most critical action.
+
+### 4. Action button (`sp-action-button`)
+
+The `variant` attribute for `sp-action-button` has been removed.
+**Action Required**: If your application uses the `variant` attribute, replace it with the `staticColor` attribute while keeping the same values. See the example below:
+
+```ts
+// Before
+<sp-action-button variant="black"></sp-action-button>
+
+// After
+<sp-action-button  staticColor="black"></sp-action-button>
```

Comment:
Not necessary to address in this PR, obviously, but I'm surprised that the syntax is `staticColor` and not `static-color`, since it seems out of line with our other APIs.

Reply:
This is the code that should be written in the application source code. You are correct though, in the DOM this will become a `static-color` attribute.

Reply:
I updated these and the `overBackground` examples :) Thank you for the call-out!